src/main/java/seedu/address/logic/commands/DeleteModuleCommand.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.ModuleCodeContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes all students identified using the inputted module code from the address book.
+ */
+public class DeleteModuleCommand extends Command {
+
+    public static final String COMMAND_WORD = "deletemodule";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes all students identified by the module code inputted.\n"
+            + "Parameters: MODULE_CODE\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_MODULE_CODE + "CS2100\n";
+
+    public static final String MESSAGE_DELETE_MULTIPLE_PERSONS_SUCCESS = "%s students deleted";
```

Comment:
```suggestion
    public static final String MESSAGE_DELETE_MULTIPLE_PERSONS_SUCCESS = "%s student(s) deleted";
```

Since one student can be deleted as well, would using "student(s)" would help? Though in the expected use case, the user probably won't `deletemodule` just for one student.